python/pyspark/sql/functions.py

```diff
@@ -1120,12 +1159,12 @@ def from_utc_timestamp(timestamp, tz):
 @since(1.5)
 def to_utc_timestamp(timestamp, tz):
     """
-    Given a timestamp, which corresponds to a certain time of day in the given timezone, returns
-    another timestamp that corresponds to the same time of day in UTC.
+    Given a `timestamp`, which corresponds to a time of day in the timezone `tz`,
```

Comment:
I had a quick question about this. The documentation seems pretty inconsistent about when to use \` vs \`\`. For example, in [approx_count_distinct](http://spark.apache.org/docs/2.1.0/api/python/pyspark.sql.html#pyspark.sql.functions.approx_count_distinct), the parameter `col` is referred to with ``, while in [rpad](http://spark.apache.org/docs/2.1.0/api/python/pyspark.sql.html#pyspark.sql.functions.rpad), the parameters are referred to with `.

Numpy appears to use single backticks for parameters:
https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections

Is there a reference for this?